src/vllm_tgis_adapter/__main__.py

```diff
@@ -274,8 +266,6 @@ async def _force_log() -> None:
 
     server = await start_grpc_server(async_llm_engine, args)
 
-    yield
-
```

Comment:
I'm assuming the `yield` was a no-op in this case?

Reply:
I think this won't work. see my comments on #9.

Perhaps something like this would work (although untested) https://gist.github.com/dtrifiro/251649959af43e3b5853bb8ab862d5a6